chat-plugins/poll.js

```diff
@@ -209,7 +209,7 @@ exports.commands = {
 					clearTimeout(room.poll.timeout);
 					room.poll.timeout = null;
 					room.poll.timeoutMins = 0;
-					return this.add("The timeout for the poll was cleared.");
+					return this.add("The timer for the poll was cleared.");
```

Comment:
Perhaps "removed" would be a better term.

lol, programmers who just reuse the names of the actual functions they use.